node-require/calculate.js

```diff
@@ -0,0 +1,19 @@
+const x = parseFloat(process.argv[2]);
+const y = parseFloat(process.argv[4]);
+const op = process.argv[3];
+const add = require('./add');
+const minus = require('./subtract');
+const multiply = require('./multiply');
+const divide = require('./divide');
```

Comment:
While it will still work how you have it, it is better practice to put all `require` statements as the first things in a file.